tests/slr_integration/__snapshots__/Editor_Test__test_get_store_data__no license - new__0.snapshot.json

```diff
@@ -0,0 +1,30 @@
+{
+    "isUsingAssignedSeating": false,
```

Comment:
Important that only here is false. It leaves the rest of the business logic unaffected just like desired